spec.html

```diff
@@ -50564,7 +50564,7 @@ <h1>
             1. Else,
               1. Assert: _asyncBody_ is an Abstract Closure with no parameters.
               1. Let _result_ be _asyncBody_().
-            1. Assert: If we return here, the async function either threw an exception or performed an implicit or explicit return; all awaiting is done.
+            1. Assert: If we return here, evaluation has finished either by throwing, by an explicit return, or by reaching the end of the Parse Node; all awaiting is done.
```

Comment:
"reaching the end of the Parse Node" is unclear to me

Reply:
Suggest an alternative wording for "got to the end of the module or function body"?

Reply:
something like "did not encounter a return, throw, or any expression that threw during Evaluation"?

Also this should be a Note, not an Assert.

Reply:
It could have done one of those things and had it swallowed by a `catch` or `finally`, so that would be wrong. Also that reads pretty awkwardly with the rest of the sentence.

This seems fine as an Assert to me: it's making explicit an invariant that would otherwise be implicit, which is exactly what Asserts are for.